### PR TITLE
Basic test environment setup

### PR DESCRIPTION
**Pull request by princiya**
* As pointed out in this issue #19, the goal is to move [non-functional code](https://github.com/electrolyfish/lightbeam-we/blob/master/js/store.js#L62) into a test directory.
* Using this [example](https://github.com/Standard8/example-webextension) as a reference, I have setup a basic test environment which includes - Karma, Mocha and Chai.
* `get` methods from `js/store.js` are now testable.
* The store object is mocked and the corresponding methods are tested.

**To test this PR:**
`npm install`
`npm run test`

**Screenshot**

test-success-1

### CODE REVIEW

File: js/store.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ const store = {
     this.addListeners();
   },
 
+  getEmptyObject() {
```
Comment (princiya) [inline js/store.js]: added for testing purpose. once I have sinon-chrome setup working, this function will be removed.

File: js/store.js [hunk 2]
```diff
@@ -59,6 +63,7 @@ const store = {
   }
 };
 
+/*
 store.getAll()
```
Comment (princiya) [inline js/store.js]: The goal of this PR is to move this code into the test directory

File: package.json [hunk 3]
```diff
@@ -4,19 +4,43 @@
...
+    "lint": "npm-run-all lint:*",
+    "lint:eslint": "eslint --ext=.js,.json .",
+    "lint:webext": "web-ext -s add-on lint",
+    "test": "npm-run-all test:*",
+    "test:lint": "npm run lint",
+    "test:karma": "karma start"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/pauljt/lightbeam-we.git"
```
Comment (princiya) [inline package.json]: haha... yeah ;)

**Comment by princiya** [comment]
I am yet to add sinon-chrome part. maybe I do that in another PR? if so, this can be merged

**Comment by groovecoder** [comment]
Nice start on the testing harness! `eslint` doesn't like a lot of stuff here, and I don't think `eslint` comes pre-configured to deal with `.json` files?

**Comment by princiya** [comment]
@groovecoder there is an `eslint-plugin-json`, `eslint-plugin-mocha` to fix the existing lint errors. #TIL

**Comment by jonathanKingston** [comment]
@princiya Lots of ESLint errors in Travis to fix:

https://travis-ci.org/electrolyfish/lightbeam-we/builds/236005297?utm_source=github_status&utm_medium=notification

Could you look into those?

**Comment by princiya** [comment]
Yes, will do. Back to DO NOT MERGE :)

File: js/background.js [hunk 4]
```diff
@@ -1,27 +1,26 @@
-'use strict';
-
-// When the user clicks browserAction icon in toolbar, run Lightbeam
-browser.browserAction.onClicked.addListener(runLightbeam);
-
-capture.init();
```
Comment (princiya) [inline js/background.js]: @biancadanforth are you able to invoke `capture.init()` from here?

Comment (biancadanforth) [inline js/background.js]: @princiya Yep. I've tested it in the browser.

**Pull request merged**